graphql_compiler/tests/test_input_data.py

```diff
@@ -448,7 +449,6 @@ def complex_optional_variables():
     # The operands in the @filter directives originate from an optional block.
     graphql_input = '''{
         Animal {
-            name @filter(op_name: "=", value: ["$animal_name"])
             out_Animal_ParentOf {
                 out_Animal_FedAt @optional {
                     name @tag(tag_name: "child_fed_at_event")
```

Comment:
It looks like the indentation here and on the line below is a bit off

Reply:
I'm not sure why this is off.